### PR TITLE
pamac: add note about AUR packages in the command description

### PR DESCRIPTION
**Pull request by Rult**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

(partially) For #5510

### CODE REVIEW

**Comment by Rult** [comment]
Yeah, that's good.

**Review by bl-ue — APPROVED** [review]
Excellent, thank you @Rult!

**Review by navarroaxel — COMMENTED** [review]
Could you change `pamac list -i` to `pamac list --installed` and `pamac remove -o` to `pamac remove --orphans`? We prefer long arguments if possible.

File: pages/linux/pamac.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # pamac
 
 > A command line utility for the GUI package manager pamac.
+> If AUR is not enabled by default, add `--aur` after a command.
```
Comment (navarroaxel) [inline pages/linux/pamac.md:4]: :thinking: is not available for every subcommand only `search`, `info` and `checkupdates`. https://gitlab.manjaro.org/applications/pamac/-/blob/master/data/doc/pamac.8.asciidoc

Comment (bl-ue) [reply pages/linux/pamac.md:4]: Hmm yeah, and `upgrade` and `update`

Comment (Rult) [reply pages/linux/pamac.md:4]: How about "If you need the AUR package(s), add --aur after a command."?

Comment (Rult) [reply pages/linux/pamac.md:4]: Or "If you don't see the AUR package(s), add --aur after a command"

**Comment by Rult** [comment]
> Could you change `pamac list -i` to `pamac list --installed` and `pamac remove -o` to `pamac remove --orphans`? We prefer long arguments if possible.
Updated to more verbose options.

File: pages/linux/pamac.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/pamac.md:4]: Hrm, if this doesn't work for all the subcommands mentioned in the tldr page, then we should probably remove this.

Comment (Rult) [reply pages/linux/pamac.md:4]: Maybe this could work?
"If you can't see the AUR packages, enable it in the GUI version or in /etc/pamac.conf"

Comment (Rult) [reply pages/linux/pamac.md:4]: *you need

Comment (navarroaxel) [reply pages/linux/pamac.md:4]: Yeah, I think is a better tip

Comment (Rult) [reply pages/linux/pamac.md]: Changed the tip.

File: pages/linux/pamac.md [hunk 2]
```diff
@@ -1,23 +1,29 @@
 # pamac
 
 > A command line utility for the GUI package manager pamac.
+> If you can't see the AUR packages, enable it in /etc/pamac.conf or in the GUI version.
```
Comment (bl-ue) [inline pages/linux/pamac.md:4]:
```suggestion
> If you can't see the AUR packages, enable it in `/etc/pamac.conf` or in the GUI version.
```

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
Welcome to tldr-pages, @Rult! 🚀

**Pull request merged**